scripts/provisioning/roles/lustre-client/tasks/main.yml

```diff
@@ -56,7 +56,9 @@
     - { version: 2.12.4, enabled: '{{ (redhat_release is version("7.7", "=")) | ternary("yes", "no") }}' }
     - { version: 2.12.5, enabled: '{{ (redhat_release is version("7.8", "=")) | ternary("yes", "no") }}' }
     - { version: 2.12.6, enabled: '{{ (redhat_release is version("7.9", "=")) | ternary("yes", "no") }}' }
-    - { version: 2.12.8, enabled: '{{ (redhat_release is version("8", ">=")) | ternary("yes", "no") }}' }
+    - { version: 2.12.6, enabled: '{{ (redhat_release is version("8.3", "=")) | ternary("yes", "no") }}' }
+    - { version: 2.12.7, enabled: '{{ (redhat_release is version("8.4", "=")) | ternary("yes", "no") }}' }
+    - { version: 2.12.8, enabled: '{{ (redhat_release is version("8.5", ">=")) | ternary("yes", "no") }}' }
```

Comment:
This will not work on Centos 8 streams.

Reply:
I'm open for suggestions. :)

Reply:
I changed it to "8" and checked on 8.5 - it works. Please check if it will work on Stream.

Reply:
- { version: 2.12.8, enabled: '{{ (redhat_release is version("8.5", ">=")) | ternary("yes", "no") }}' }
+    - { version: 2.12.8, enabled: '{{ (redhat_release is version("8", "=")) | ternary("yes", "no") }}' }

This change is worked and i think the change you have done too will work.

Reply:
Not sure this will work on 8.5.